inc/3rd-party/hosting/litespeed.php

```diff
@@ -0,0 +1,67 @@
+<?php
+use WP_Rocket\Logger\Logger;
+
+defined('ABSPATH') || die('Cheatin&#8217; uh?');
+
+/**
+ * Allow to purge the Litespeed cache
+ *
+ * @since 3.4.1
+ * @author Soponar Cristina
+ */
+if ( isset( $_SERVER['HTTP_X_LSCACHE'] ) && $_SERVER['HTTP_X_LSCACHE'] ) {
+    /**
+     * Purge all the domain
+     *
+     * @since 3.4.1
```

Comment:
Sorry, those are still missing.

```suggestion
     * @since 3.4.1
     * @author Soponar Cristina
```

Reply:
I was blind for these missing `@author `
Hope that now are fine.

Reply:
![yes](https://user-images.githubusercontent.com/1878479/66518405-769e3180-eae5-11e9-9212-783b73d64ae0.jpg)